src/consumption/elevate.js

```
@@ -0,0 +1,848 @@
+			await resourceService.publishCallback(projectData.id, templateId.toString())
+
+			//return result
+			result.success = true
+			result.templateId = templateId
+			return resolve(result)
+		} catch (error) {
+			if (mongoConnection) mongoConnection.disconnect()
+			result.error = error.message || error
+			return reject(error)
+		}
+	})
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)